helm/botkube/values.yaml

```diff
@@ -200,6 +200,9 @@ resources: {}
   #  cpu: 100m
   #  memory: 128Mi
 
+extraEnv:   # Extra environment variables to pass to the botkube container, example HTTP_PROXY
+  #HTTP_PROXY: <proxyURL>:<port>
+  
```

Comment:
@rajinator please move all env vars under `extraEnv` section - `CONFIG_PATH`, `LOG_LEVEL`

Reply:
Are these two considered optional? Or mandatory? If mandatory, makes sense to add another section called `env`, so they don't get ignored

Reply:
I think we should not move other variables to values.yaml. 
1. `LOG_LEVEL` is already configurable via different variable. 
2. If we want to make `CONFIG_PATH` configurable through values.yaml then we have to take care of volume mount as well. (Does that make sense to have this variable configurable by the user as BotKube binary already runs inside a Pod (container), so the path should not be an issue to someone)

It would be better to keep these variables unexposed to users.

Reply:
Seems it's ready to merge then? @bhavin192

Reply:
Ok. For now, we have only `LOG_LEVEL` configurable variable. As suggested, in future, if we have multiple vars, let's keep them in `env` section